src/lotus/client.rs

```diff
@@ -59,6 +61,7 @@ mod methods {
     pub const IPC_LIST_BOTTOMUP_CHECKPOINTS: &str = "Filecoin.IPCListCheckpointsSerialized";
     pub const IPC_GET_TOPDOWN_MESSAGES: &str = "Filecoin.IPCGetTopDownMsgsSerialized";
     pub const IPC_GENESIS_EPOCH_FOR_SUBNET: &str = "Filecoin.IPCGetGenesisEpochForSubnet";
+    pub const IPC_HAS_VOTED_BOTTOM_UP_CHECKPOINT: &str = "Filecoin.IPCHasVotedBottomUpCheckpoint";
```

Comment:
Wasn't this endpoint already implemented? (I may be wrong)

Reply:
yeah, removed already